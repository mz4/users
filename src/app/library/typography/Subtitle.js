import classnames from 'classnames';
import styles from './Subtitle.module.scss';

export const Subtitle = ({ text, customClassName = '' }) => {
  return (
    <div className={classnames(styles.pageSubTitle, customClassName)}>
      <span>{text}</span>
    </div>
  );
};
