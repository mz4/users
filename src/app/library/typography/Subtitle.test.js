import React from 'react';
import renderer from 'react-test-renderer';
import { Subtitle } from './Subtitle';

describe('Render subtitle component', () => {
  it('Create a snapshot', () => {
    const loader = <Subtitle />;
    const component = renderer.create(loader);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
