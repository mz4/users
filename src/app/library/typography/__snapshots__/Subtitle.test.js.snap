// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render subtitle component Create a snapshot 1`] = `
<div
  className="pageSubTitle"
>
  <span />
</div>
`;
